templates/cloud/kubernetes.html

```diff
@@ -0,0 +1,116 @@
+        <h2>Production features</h2>
+        <ul class="list-ticks--compact">
+            <li>Access to the latest stable upstream Kubernetes release</li>
+            <li>Access to daily ‘edge’ builds of upstream Kubernetes development branch (1.5.x) with rollback</li>
+            <li>Scale master nodes independently of worker nodes</li>
+            <li>Workloads are automatically portable between public clouds, private clouds and bare metal with an easy onramp to Google Container Engine (GKE)</li>
+            <li>Easily create and remove user access</li>
+            <li>Maintenance mode and supported upgrade path to new upstream versions</li>
+            <li>Kubernetes dashboard, Prometheus, for monitoring and alerting, collection and processing of system metrics</li>
+            <li>Elastic Stack for Insights with Beats log collection and monitoring and analysis and visualisation with Elasticsearch and Kibana</li>
+            <li>Optionally backed by Ceph for filesystem and block storage</li>
+        </ul>
+    </div>
+    <div class="juju-bundle-card six-col last-col">
+        <div class="juju-card" data-id="canonical-kubernetes"></div>
+    </div>
+</section>
+
+<section class="row row-grey">
+    <div class="equal-height">
+        <div class="four-col equal-height__item equal-height__align-vertically not-for-small">
+            <img src="{{ ASSET_SERVER_URL }}9f035e4a-picto-support-orange.svg" width="200" alt="" />
+        </div>
+        <div class="eight-col last-col equal-height__item">
+            <h2>Support and customisation services</h2>
+            <p>Enterprise support for Kubernetes is provided by Canonical in partnership with Google where customers gain access to a global pool of knowledge and expertise.</p>
+            <p>We also offer consulting services and customisation for larger organisations to integrate Kubernetes with existing infrastructure.</p>
+            <p>For customers needing a fully managed Kubernetes, Canonical will deploy and operate your cluster remotely, and hand over to your own team as soon as they are familiar with the operational practices under the hood.</p>
+            <p>To discuss your requirement connect with a member of the Canonical team.</p>
+            <p><a href="/cloud/contact-us" class="button--primary">Contact us&nbsp;&rsaquo;</a></p>
+        </div>
+    </div>
+</section>
+
+<section class="row row-expertise no-border">
+    <div class="equal-height">
+        <div class="six-col equal-height__item">
+            <h2>Exchange expertise and ideas with thousands of other IT professionals</h2>
+            <p>Want to talk to other Ubuntu users straight away? Share ideas and get advice and help from our large, active community of IT professionals. As a community, we set high standards for friendliness and tolerance. We welcome your questions and contributions!</p>
+            <p><a href="/support/community" class="external">Community support</a></p>
```

Comment:
it is an external link if you follow it. That's a redirect. I'll flag that and confirm that it needs to be